.github/workflows/benchmarks.yml

```diff
@@ -92,7 +92,9 @@ jobs:
         with:
           r-version: ${{ matrix.r-version }}
       - name: "Set up LuaJit"
-        uses: leafo/gh-actions-lua@v8.0.0
+        run: |
+          cd ~/work/Microbenchmarks/Microbenchmakrs/lua
```

Comment:
I think this is the issue^

Also, why not use the github action version? It would let us specify the version explicitly.

Reply:
The reason we can't use the github action is because of how it is recommended to install scilua. See the Install section under their homepage: https://scilua.org/

Reply:
Got it, thanks! Fine with me then. It also looks like that github action isn't consistently maintained anyways.

Reply:
Btw that directory should be fixed `Microbenchmakrs` => `Microbenchmarks`

Reply:
omg ty so much let me fix and try to get it working